hicommonwealth/commonwealth
# Cosmos linking additions

## Link to Issue
Closes: #3674 

## Description of Changes
- Adds `ProposalSelector` component to `update_proposal_status_modal.tsx` to replace `chainEntitySelector` for cosmos proposals
- unique cosmos link builder
- Changes to pull cosmos proposals through to the modal

## Test Plan
- Ensure expected cosmos proposals render in modal
- Select proposal and ensure links appear on thread
- delete links 


## Other Considerations

- There was an issue with clicking the link to the proposal and 'Proposal Not Found' which was determined to be out of scope for this PR
